integration-tests/tests/basic-happy-path.spec.ts

```diff
@@ -1,43 +1,38 @@
 import { applicationDetailPagePO } from '../support/pageObjects/createApplication-po';
 import { AddComponentPage } from '../support/pages/AddComponentPage';
 import { ApplicationDetailPage } from '../support/pages/ApplicationDetailPage';
-import { PipelinerunsTabPage } from '../support/pages/tabs/PipelinerunsTabPage';
+import { DetailsTab, PipelinerunsTabPage } from '../support/pages/tabs/PipelinerunsTabPage';
 import { Applications } from '../utils/Applications';
 import { Common } from '../utils/Common';
+import { UIhelper } from '../utils/UIhelper';
 
-describe('Create Components using the UI', { tags: ['@PR-check', '@publicRepo'] }, () => {
+describe('Basic Happy Path', { tags: ['@PR-check', '@publicRepo'] }, () => {
   const applicationName = Common.generateAppName();
   const applicationDetailPage = new ApplicationDetailPage();
   const pipelinerunsTab = new PipelinerunsTabPage();
   const addComponent = new AddComponentPage();
-  const publicRepos = [
-    'https://github.com/hac-test/devfile-sample-code-with-quarkus',
-  ];
-  const componentNames: string[] = ['java-quarkus'];
-  const deploymentBody = new Map<string, string>();
+  const publicRepo = 'https://github.com/hac-test/devfile-sample-code-with-quarkus';
+  const componentName: string = Common.generateAppName("java-quarkus");
+  const piplinerunlogsTasks = ["init", "clone-repository", "build-container", "show-summary"]
   const quarkusDeplomentBody = 'Congratulations, you have created a new Quarkus cloud application';
+  const deploymentBody = new Map<string, string>([[componentName, quarkusDeplomentBody]]);
```

Comment:
the map looks redundant now, you can just address `componentName` and `quarkusDeplomentBody` directly